src/storage/hummock_test/src/failpoint_tests.rs

```diff
@@ -106,9 +103,6 @@ async fn test_failpoints_state_store_read_upload() {
     // sync epoch1 test the read_error
     let ssts = hummock_storage.sync(1).await.unwrap().uncommitted_ssts;
     meta_client.commit_epoch(1, ssts).await.unwrap();
-    local_version_manager
```

Comment:
How to ensure the latest version is pushed? Same below.

Reply:
> ensure the latest version is pushed

why do we need to ensure the latest version is pushed in this test?

Reply:
> How to ensure the latest version is pushed? Same below.

fixed